R/HDF5AnnData.R

```diff
@@ -36,6 +38,24 @@ HDF5AnnData <- R6::R6Class("HDF5AnnData", # nolint
         write_h5ad_element(value, private$.h5obj, "/layers")
       }
     },
+    #' @field obsm
+    obsm = function(value) {
+      if (missing(value)) {
+        read_h5ad_element(private$.h5obj, "obsm")
+      } else {
+        # TODO: validate obsm
+        write_h5ad_element(value, private$.h5obj, "/obsm")
+      }
+    },
+    #' @field varm
+    varm = function(value) {
+      if (missing(value)) {
+        read_h5ad_element(private$.h5obj, "varm")
+      } else {
+        # TODO: validate varm
```

Comment:
Add trackstatus comment

```suggestion
        # trackstatus: class=HDF5AnnData, feature=set_varm, status=wip
        # TODO: validate varm
```